docs/src/configuring-zed.md

```diff
@@ -1731,6 +1731,44 @@ Run the `theme selector: toggle` action in the command palette to see a current
 }
 ```
 
+## Assistant Panel
+
+- Description: Customize assistant panel
+- Setting: `assistant`
+- Default:
+
+```json
+"assistant": {
+  "enabled": true,
+  "button": true,
+  "dock": "right",
+  "default_width": "N/A - width in pixels",
```

Comment:
These width and height settings have an actual pixel value as the default. Can we adjust this to show that value?